backend/src/entities/book.ts

```diff
@@ -6,21 +6,36 @@ export class Book {
   @PrimaryGeneratedColumn()
   id!: number;
 
+  @Column({ nullable: true })
+  userId!: number;
+
   @Column()
   title!: string;
 
   @Column()
-  desc!: string;
+  age!: string;
 
   @Column()
-  author!: string;
+  subject!: string;
+
+  @Column({ nullable: true })
+  charName!: string;
+
+  @Column({ nullable: true })
+  charDesc!: string;
```

Comment:
Actually, we don't need to store some of the inputs because they are only used as query descriptions for OpenAI to create the book. If we need to store them in that case, I should create another entity table to store the characters and the character name in a separate database table